src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public x: number,
+    public y: number,
+    public z: number,
```

Comment:
Try to use a, b, c... in math tasks, because x, y, z is a coords

Reply:
will be better use `a, b, c` in this case